stdlib/tkinter/__init__.pyi

```diff
@@ -949,28 +949,28 @@ class Toplevel(BaseWidget, Wm):
         cnf: dict[str, Any] | None = {},
         *,
         background: str = ...,
-        bd: _ScreenUnits = ...,
+        bd: _ScreenUnits = 0,
         bg: str = ...,
         border: _ScreenUnits = ...,
-        borderwidth: _ScreenUnits = ...,
-        class_: str = ...,
-        colormap: Literal["new", ""] | Misc = ...,
-        container: bool = ...,
-        cursor: _Cursor = ...,
-        height: _ScreenUnits = ...,
+        borderwidth: _ScreenUnits = 0,
+        class_: str = "Toplevel",
+        colormap: Literal["new", ""] | Misc = "",
+        container: bool = False,
+        cursor: _Cursor = "",
+        height: _ScreenUnits = 0,
         highlightbackground: str = ...,
         highlightcolor: str = ...,
-        highlightthickness: _ScreenUnits = ...,
+        highlightthickness: _ScreenUnits = 0,
         menu: Menu = ...,
         name: str = ...,
-        padx: _ScreenUnits = ...,
-        pady: _ScreenUnits = ...,
-        relief: _Relief = ...,
-        screen: str = ...,  # can't be changed after creating widget
-        takefocus: _TakeFocusValue = ...,
+        padx: _ScreenUnits = "0",
+        pady: _ScreenUnits = "0",
```

Comment:
These appear to be `_tkinter.Tcl_Obj` instances by default, so the string `"0"` is no more accurate than an integer `0`:

```
>>> top['padx']
<pixel object: '0'>
>>> top['pady']
<pixel object: '0'>
```

IMO we should just use the integer `0` when there is no padding or border or similar, as `0`, `"0"` and `<pixel object: '0'>` are equivalent anyway. An integer is probably the most useful thing for IDE autocompletions and such.

Reply:
I agree that an integer is probably the most useful.

(Although on Windows, `padx` and `pady` are sometimes `<string object: '0'>` and sometimes just `'0'` - this inconsistency is due to python/cpython#101830):
```
>>> import tkinter
>>> root = tkinter.Tk()
>>> top = tkinter.Toplevel(root)
>>> top['padx']
<string object: '0'>
>>> top['padx']
'0'
>>> top['pady']
<string object: '0'>
>>> top['pady']
'0'
```

Reply:
> For the rest of the integer-in-strings, the default value is consistently a string on all platforms,
so I'm not sure if I should replace those with a plain integer (this would probably be more useful in IDEs but less accurate)

IMO we should just use ints. If we sometimes use `"0"` and sometimes `0`, it will probably make people wonder what their difference is, even though there is no practical difference.